### DISCUSSION TITLE
Windows 11 context menu full crashes Explorer

### DISCUSSION THREAD

**Discussion by jamesburt** [valinet/ExplorerPatcher#2464] [General]
EP: 22621.2506.60.1
OS Name	Microsoft Windows 11 Home
Version	10.0.22621 Build 22621

Just patched windows and updated EP afterwards.

I use the win11 context menu, anytime I right click file explore folders or file objects, explorer complete crashes out of memory, no prompts, straight to gone, and then relaunches itself.

For now I've gone ahead and disabled win11 context menu. Things seem stable.
Prior to this windows update and EP update, system has run fine without crashes.

Faulting application name: explorer.exe, version: 10.0.22621.2428, time stamp: 0xaab5a73e
Faulting module name: ntdll.dll, version: 10.0.22621.2428, time stamp: 0x806d3ecf
Exception code: 0xc0000374
Fault offset: 0x000000000010c849
Faulting process id: 0x0x250C
Faulting application start time: 0x0x1DA1A18B2A560BF
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: 864e70b4-7b06-4798-ab7a-524b37e491f3
Faulting package full name: 
Faulting package-relative application ID: 

Faulting application name: explorer.exe, version: 10.0.22621.2428, time stamp: 0xaab5a73e
Faulting module name: ntdll.dll, version: 10.0.22621.2428, time stamp: 0x806d3ecf
Exception code: 0xc0000374
Fault offset: 0x000000000010c849
Faulting process id: 0x0xC24
Faulting application start time: 0x0x1DA1A189FAF04C8
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: 1a4ff742-868f-4ef0-a930-556346ddf219
Faulting package full name: 
Faulting package-relative application ID: 

Faulting application name: explorer.exe, version: 10.0.22621.2428, time stamp: 0xaab5a73e
Faulting module name: ntdll.dll, version: 10.0.22621.2428, time stamp: 0x806d3ecf
Exception code: 0xc0000374
Fault offset: 0x000000000010c849
Faulting process id: 0x0x194
Faulting application start time: 0x0x1DA1A181F8EEA20
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: c808efc6-01d5-472f-a175-dc51ab172b00
Faulting package full name: 
Faulting package-relative application ID: 

One of the event details from event logs:
- System 

  - Provider 

   [ Name]  Application Error 
   [ Guid]  {a0e9b465-b939-57d7-b27d-95d8e925ff57} 
 
   EventID 1000 
 
   Version 0 
 
   Level 2 
 
   Task 100 
 
   Opcode 0 
 
   Keywords 0x8000000000000000 
 
  - TimeCreated 

   [ SystemTime]  2023-11-18T12:10:13.8818040Z 
 
   EventRecordID 102590 
 
   Correlation 
 
  - Execution 

   [ ProcessID]  33268 
   [ ThreadID]  21328 
 
   Channel Application 
 
   Computer redacted
 
  - Security 

   [ UserID]  redacted 
 

- EventData 

  AppName explorer.exe 
  AppVersion 10.0.22621.2428 
  AppTimeStamp aab5a73e 
  ModuleName ntdll.dll 
  ModuleVersion 10.0.22621.2428 
  ModuleTimeStamp 806d3ecf 
  ExceptionCode c0000374 
  FaultingOffset 000000000010c849 
  ProcessId 0x194 
  ProcessCreationTime 0x1da1a181f8eea20 
  AppPath C:\WINDOWS\explorer.exe 
  ModulePath C:\WINDOWS\SYSTEM32\ntdll.dll 
  IntegratorReportId c808efc6-01d5-472f-a175-dc51ab172b00 
  PackageFullName  
  PackageRelativeAppId  

  
  I'm providing this more as a FYI, not sure if I'll remember to follow this conversation at all.

**Comment by pyrates999** [comment]
Check what happens when using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.

**Reply by jamesburt** [reply]
the console poofs with the rest of the explorer/taskbar entities
tried recording the screen, but no errors displayed as they all poof


rclick any file object in file explorer then get the spinning blue donut for a few seconds then **POOF** to everything explorer related

**Comment by pyrates999** [comment]
Your windows build is incomplete.  Please post windows version and build.  You can find this by running winver.

**Reply by jamesburt** [reply]
<img width="336" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/2118555/4676c95a-3a5e-4b48-85e9-8cab6e6f8394">

**Reply by jamesburt** [reply]
with win 11 context disabled, when rclick file objects in explorer windows, the png and empty  menu errors come up for each rclick

<img width="778" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/2118555/cd0a695c-efab-4607-9c2c-2adbea472cb1">

**Reply by pyrates999** [reply]
Please show a screen shot of the option in EP that you think is causing this.

**Reply by jamesburt** [reply]
I said what the issue is.
Not having win 11 context menu disabled, disable the win 11 context, and I get the old style context and it doesn't crash on rclick, enable win 11 context, and back to instant poof crashes of everything tied to explorer.exe
I have used this this flawlessly for longer than I can remember with the the win 11 context enabled, then I patched windows & ep yesterday and now the crashes

**Reply by pyrates999** [reply]
A screen shot is more clear and easy to understand.

**Comment by pyrates999** [comment]
I think this is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2218

please close this
